### DISCUSSION TITLE
Table of Contents File - Add Last Updated Date Field

### DISCUSSION THREAD

**Discussion by jonberkery-work** [CMSgov/price-transparency-guide#558] [Ideas]
I would like to suggest that the Table of Contents (TOC) File include a Last Updated Date field to the schema.  With caching setting so diverse in these solutions, it is time consuming to identify what TOC file is actually consumed, especially as we enter month 2.  I would like to suggest the last updated field be at the "Top level" only, like the in-network-rates and allowed-amounts schemas have.  I would also like to suggest the field be optional, so that all file providers can implement it at their own cadence.

What does the community think about this suggestion?

**Comment by shaselton-usds** [comment]
@jonberkery-work Following up on some older threads -- this makes sense, but could you do the same thing by simply calculating a hash and then comparing the values?
